examples/react-alt/js/actions/todoActions.js

```diff
@@ -0,0 +1,35 @@
+/*jshint quotmark:false */
+/*jshint white:false */
+/*jshint trailing:false */
```

Comment:
`trailing` and `white` are [deprecated](https://github.com/jshint/jshint/releases/tag/2.5.0).